components/shared/badge/Badge.tsx

```diff
@@ -1,20 +1,18 @@
+import { type VariantProps, cva } from "class-variance-authority"
 import * as React from "react"
-import { cva, type VariantProps } from "class-variance-authority"
 
 import { cn } from "@/lib/helpers/helper.cn"
 
 const badgeVariants = cva(
-  "inline-flex items-center rounded-md border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2",
+  `inline-flex items-center rounded-full px-2 py-1 text-typo-tag-sm font-semibold transition-colors
+  text-background`,
   {
     variants: {
       variant: {
-        default:
-          "border-transparent bg-primary text-primary-foreground shadow hover:bg-primary/80",
-        secondary:
-          "border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80",
-        destructive:
-          "border-transparent bg-destructive text-destructive-foreground shadow hover:bg-destructive/80",
-        outline: "text-foreground",
+        default: "bg-foreground text-background",
+        outline: "bg-secondary border-1 border border-foreground text-foreground",
+        destructive: "bg-destructive text-destructive-foreground",
+        blue: "bg-blue-title",
```

Comment:
Consider either calling the blue variable something not tied to blue titles or specific to blue titles

```suggestion
        blue: "bg-blue",
```
```suggestion
        "blue-title": "bg-blue-title",
```